src/EntryPoints/PageApprovalsHooks.php

```diff
@@ -17,4 +20,46 @@ public static function onOutputPageParserOutput( OutputPage $out, ParserOutput $
 		);
 	}
 
+	public static function onOutputPageBeforeHTML( OutputPage $out ): void {
+		$user = $out->getUser();
+		$title = $out->getTitle();
```

Comment:
Better to call these when you need them

https://www.entropywins.wtf/blog/2018/10/24/readable-functions-minimize-state/